testing/py-apipkg/APKBUILD

```diff
@@ -0,0 +1,53 @@
+# Contributor: Dmitry Romanenko <dmitry@romanenko.in>
+# Maintainer: Dmitry Romanenko <dmitry@romanenko.in>
+pkgname=py-apipkg
+_pkgname=apipkg
+pkgver=1.5
+pkgrel=0
+pkgdesc="apipkg: namespace control and lazy-import mechanism"
+url="https://github.com/pytest-dev/apipkg"
+arch="noarch"
+license="MIT"
+options="!check" # Tests are outdated
+depends=""
+checkdepends="py-py pytest py-tox"
+makedepends="python2-dev python3-dev py-setuptools py-setuptools_scm"
+subpackages="py3-$_pkgname:_py3 py2-$_pkgname:_py2"
+source="https://files.pythonhosted.org/packages/source/${_pkgname:0:1}/$_pkgname/$_pkgname-$pkgver.tar.gz"
+builddir="$srcdir"/$_pkgname-$pkgver
+
+build() {
+	cd "$builddir"
+	python2 setup.py build
+	python3 setup.py build
+}
+
+check() {
+	cd "$builddir"
+	tox -e py27,py36
```

Comment:
why 36 used when 3.7 in repo?

Reply:
Oops, going to update it shortly

Reply:
I actually took a look and didn't find py37.
tox.ini contains py36 only, as well as Alpine repo simply has only Python 3.6.7
https://github.com/pytest-dev/apipkg/blob/master/tox.ini

Reply:
Btw ehst about using `pytest` here instead of tox,flakes, like https://github.com/pytest-dev/apipkg/blob/master/.travis.yml#L19

Reply:
I checked and found no way to run tests... please remove the `check()` at all and `checkdepends` line

Reply:
I was thinking about keeping tests inplace but disabling them (until it actually gets fixed in their repo... ever?)
But i'm okay to remove them to avoid noise.